x-pack/plugins/observability_solution/slo/public/pages/slo_details/components/overview/overview.tsx

```diff
@@ -170,6 +169,19 @@ export function Overview({ slo }: Props) {
             }
           />
         )}
+
+        <OverviewItem
+          title={i18n.translate('xpack.slo.sloDetails.overview.settings.syncDelay', {
+            defaultMessage: 'Sync delay',
+          })}
+          subtitle={slo.settings.syncDelay}
+        />
+        <OverviewItem
+          title={i18n.translate('xpack.slo.sloDetails.overview.settings.frequency', {
+            defaultMessage: 'Frequency',
+          })}
+          subtitle={slo.settings.frequency}
+        />
```

Comment:
I'm not spending too much time here since Maciej has some plan for the new overview section